### PR TITLE
Refactor / fix addinfo

### PR DESCRIPTION
**Pull request by PtrckM**
**Before submitting this PR, please make sure**:

- [x] Your code builds clean without any errors or warnings
- [x] You are using approved title ("feat/", "fix/", "docs/", "refactor/")

**After creating this PR, please make sure:**

- [x] You link the related GitHub issue or ticket https://github.com/CoinAlpha/hummingbot/issues/2616

**A description of the changes proposed in the pull request**:
- fix extra bracket
- add additional info when editing the config (request from user on [discord](https://discord.com/channels/530578568154054663/774726042891321404/785328131233939476))
image
image

### CODE REVIEW

File: hummingbot/templates/conf_perpetual_market_making_strategy_TEMPLATE.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ market: null
 # What leverage to used
 leverage: null
 
-# Position mode to use, one-way or hedge position mode
+# Position mode to use, one-way or hedge position mode (If disabled, one-way position mode would be used. (True or False))
```
Comment (Nullably) [inline hummingbot/templates/conf_perpetual_market_making_strategy_TEMPLATE.yml:17]: This is unclear on what values are expected for this parameter. Is it one-way/hedge or True/False ?

**Review by Nullably — CHANGES_REQUESTED** [review]

File: hummingbot/templates/conf_perpetual_market_making_strategy_TEMPLATE.yml (hunk 1 above)
Comment (PtrckM) [reply hummingbot/templates/conf_perpetual_market_making_strategy_TEMPLATE.yml]: I update the additional info.

**Review by Nullably — APPROVED** [review]
LGTM

File: hummingbot/templates/conf_perpetual_market_making_strategy_TEMPLATE.yml [hunk 2]
```diff
@@ -14,7 +14,8 @@ market: null
 # What leverage to used
 leverage: null
 
-# Position mode to use, one-way or hedge position mode
+# Position mode to use, hedge mode or one-way position mode (true/false).
+# If value is set to true, hedge mode is enabled.
```
Comment (vic-en) [inline hummingbot/templates/conf_perpetual_market_making_strategy_TEMPLATE.yml:18]:
```suggestion
# Position mode to use, hedge mode or one-way position mode.
```
I addressed this issue in the PR to add position management. So users now have to specify either "One-way" or "Hedge" for position mode rather that True/False.

**Review by vic-en — CHANGES_REQUESTED** [review]

**Review by vic-en — APPROVED** [review]
LGTM

**Pull request merged**